dev/tests/api-functional/testsuite/Magento/GraphQl/Review/GetProductReviewsTest.php

```diff
@@ -0,0 +1,284 @@
+                'values' => [
+                    [
+                        'value_id' => 'Ng==',
+                        'value' => "1"
+                    ],[
+                        'value_id' => 'Nw==',
+                        'value' => "2"
+                    ],[
+                        'value_id' => 'OA==',
+                        'value' => "3"
+                    ],[
+                        'value_id' => 'OQ==',
+                        'value' => "4"
+                    ],[
+                        'value_id' => 'MTA=',
+                        'value' => "5"
+                    ]
+                ]
+            ]
+        ];
+        $response = $this->graphQlQuery($query);
+        self::assertArrayHasKey('productReviewRatingsMetadata', $response);
+        self::assertArrayHasKey('items', $response['productReviewRatingsMetadata']);
+        self::assertNotEmpty($response['productReviewRatingsMetadata']['items']);
+        self::assertEquals($expectedRatingItems, $response['productReviewRatingsMetadata']['items']);
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/Review/_files/different_reviews.php
+     */
+    public function testProductReviewRatings()
+    {
+        $productSku = 'simple';
+        /** @var ProductRepositoryInterface $productRepository */
+        $productRepository = ObjectManager::getInstance()->get(ProductRepositoryInterface::class);
+        $product = $productRepository->get($productSku, false, null, true);
+        $summaryFactory = ObjectManager::getInstance()->get(SummaryFactory::class);
+        $storeId = ObjectManager::getInstance()->get(StoreManagerInterface::class)->getStore()->getId();
+        $summary = $summaryFactory->create()->setStoreId($storeId)->load($product->getId());
+        $query
+            = <<<QUERY
+{
+  products(filter: {
+      sku: {
+          eq: "$productSku"
+      }
+  }) {
+    items {
+      rating_summary
+      review_count
+      reviews {
+        items {
+          nickname
+          summary
+          text
+          average_rating
+          product {
+            sku
+            name
+          }
+          ratings_breakdown {
+            name
+            value
+          }
+        }
+      }
+    }
+  }
+}
+QUERY;
+        $response = $this->graphQlQuery($query);
+        self::assertArrayHasKey('products', $response);
+        self::assertArrayHasKey('items', $response['products']);
+        self::assertNotEmpty($response['products']['items']);
+
+        $items = $response['products']['items'];
+        self::assertEquals($summary->getData('rating_summary'), $items[0]['rating_summary']);
+        self::assertEquals($summary->getData('reviews_count'), $items[0]['review_count']);
+        self::assertArrayHasKey('items', $items[0]['reviews']);
+        self::assertNotEmpty($items[0]['reviews']['items']);
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/Review/_files/customer_review_with_rating.php
```

Comment:
@eduard13 This is the reason why the web api test is failing. customer_review_with_rating.php doesn't have a rollback. When using existing fixtures look out for cleanups since some aren't rolled back properly. Adding one to this should unblock you.

Reply:
Thank you for the hint, it's really appreciated.

Reply:
Hi @prabhuram93, looks like adding rollbacks have fixed the issues. 
Could you please continue reviewing this one?
Thank you.